heartbeat/podman

```diff
@@ -376,6 +376,11 @@ run_new_container()
 		# with the same name, even though podman itself thinks there
 		# is no such container. If so, purge the storage layer to try
 		# to clean the corruption and try again.
+                if echo "$out" | grep -q "unknown.*flag"; then
+                        ocf_exit_reason "$out"
+                        return $rc
+                fi
```

Comment:
Can you update the code to use tabs instead of spaces for indentation, so it follows the rest of the agent?

Reply:
ok , wait a minute, let me replace it